google/ggrc-core
# GGRC-4505 Deny CycleTask creation for some types

# Issue description
user shouldn't be able to create CycleTask and automatically map to next types via global search: Workflow, TaskGroup, CycleTask, TaskGroupTask, Person.

# Steps to test the changes
1. Open Global Search modal
2. Search for particular object type
3. Click on a row in search results
4. Open 3bb's menu
"Create Task" should be shown for expected object types only.

# Solution description
For now list of mappable to CycleTask objects is stored on front-end side. It is needed to store it on BE and provide it in GGRC.config object.

# Sanity checklist


# PR Review checklist
